client/ayon_zbrush/plugins/create/create_workfile.py

```diff
@@ -45,9 +45,10 @@ def create(self):
                 host_name,
             )
             data = {
-                "task": task_entity,
+                "task": task_name,
                 "variant": variant,
-                "folderPath": folder_path
+                "folderPath": folder_path,
+                "taskType": task_entity["taskType"],
```

Comment:
Hey @BigRoy,

I added a `taskType` to prevent a key error I was seeing when running the publisher app.  I'm doing a little more testing at the moment to ensure it's still needed. I should have a pretty good idea at some point tomorrow.

Reply:
@BigRoy It does appear to be a non-issue at the moment. I will remove it, but keep an eye out if the issue arises again in production.